browser/content_settings/brave_content_settings_registry_unittest.cc

```diff
@@ -185,30 +185,38 @@ TEST_F(BraveContentSettingsRegistryTest, GetInitialDefaultSetting) {
     EXPECT_EQ(CONTENT_SETTING_ASK, info->GetInitialDefaultSetting());
   }
 
-  // TODO(bridiver) - DEFAULT is not a valid setting, but leaving it for
-  // now because this requires more extensive changes to fix correctly
-  // {
-  //   SCOPED_TRACE("Content setting: BRAVE_FINGERPRINTING_V2");
-  //   info = registry()->Get(ContentSettingsType::BRAVE_FINGERPRINTING_V2);
-  //   EXPECT_EQ(CONTENT_SETTING_DEFAULT, info->GetInitialDefaultSetting());
-  // }
+  {
+    SCOPED_TRACE("Content setting: BRAVE_FINGERPRINTING_V2");
+    info = registry()->Get(ContentSettingsType::BRAVE_FINGERPRINTING_V2);
+    EXPECT_EQ(CONTENT_SETTING_ASK, info->GetInitialDefaultSetting());
+  }
 
-  // {
-  //   SCOPED_TRACE("Content setting: BRAVE_COSMETIC_FILTERING");
-  //   info = registry()->Get(ContentSettingsType::BRAVE_COSMETIC_FILTERING);
-  //   EXPECT_EQ(CONTENT_SETTING_DEFAULT, info->GetInitialDefaultSetting());
-  // }
+  {
+    SCOPED_TRACE("Content setting: BRAVE_COSMETIC_FILTERING");
+    const WebsiteSettingsInfo* ws_info = website_settings_registry()->Get(
+        ContentSettingsType::BRAVE_COSMETIC_FILTERING);
+    const auto initial_value =
+        brave_shields::CosmeticFilteringSetting::FromValue(
+            ws_info->initial_default_value());
+    EXPECT_EQ(brave_shields::ControlType::BLOCK_THIRD_PARTY, initial_value);
+    EXPECT_EQ(brave_shields::ControlType::BLOCK_THIRD_PARTY,
+              brave_shields::CosmeticFilteringSetting::kDefaultValue);
+    EXPECT_EQ(ws_info->initial_default_value(),
+              brave_shields::CosmeticFilteringSetting::DefaultValue());
+  }
 
-  // {
-  //   SCOPED_TRACE("Content setting: BRAVE_COOKIES");
-  //   info = registry()->Get(ContentSettingsType::BRAVE_COOKIES);
-  //   EXPECT_EQ(CONTENT_SETTING_DEFAULT, info->GetInitialDefaultSetting());
-  // }
+  {
+    SCOPED_TRACE("Content setting: BRAVE_COOKIES");
```

Comment:
why is this being changed? This seems independent of BRAVE_COSMETIC_FILTERING changes

Reply:
then it should be split out into a separate PR. We shouldn't be putting unrelated changes in PRs just for convenience